Products/PluggableAuthService/plugins/DynamicGroupsPlugin.py

```diff
@@ -331,6 +333,8 @@ def addGroup(self, group_id, predicate, title='', description='',
         view_name = createViewName('enumerateGroups')
         self.ZCacheable_invalidate(view_name=view_name)
 
+        notify(GroupCreated(group_id))
+
```

Comment:
Needs new test assertion that the event is emitted.

Reply:
Hi @tseaver 

how can I do this? I can either register a testing subscriber that "does something", but I suppose there is another way to check this, do you have sample test somewhere?

Thank you for feedback,

Gauthier

Reply:
This could be an idea to write such a test.